cli/internal/cmd/iam.go

```diff
@@ -353,6 +353,7 @@ func (c *awsIAMCreator) printOutputValues(cmd *cobra.Command, flags iamFlags, ia
 	cmd.Printf("zone:\t\t\t%s\n", flags.aws.zone)
 	cmd.Printf("iamProfileControlPlane:\t%s\n", iamFile.AWSOutput.ControlPlaneInstanceProfile)
 	cmd.Printf("iamProfileWorkerNodes:\t%s\n\n", iamFile.AWSOutput.WorkerNodeInstanceProfile)
+	cmd.Println("Your IAM configuration was created successfully. Please fill the above values into your configuration file.")
```

Comment:
```suggestion
```
This message is already printed by the [main `create` function](https://github.com/edgelesssys/constellation/blob/18a6007356dbde4789a51f016f6dd68b88cc0e98/cli/internal/cmd/iam.go#L229).
Instead of also adding the output to the AWS and GCP IAM creator, it should be removed from the Azure creator